Experiments/forest_QKS/qks.py

```diff
@@ -0,0 +1,223 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.utils import shuffle
+
+from pyquil import get_qc
+from pyquil.quil import Program
+from pyquil.gates import RX, CNOT, MEASURE
+
+
+def make_frames(num):
+    """
+    The picture frames dataset contains a smaller inner square
+    with sides of length 2 and a larger outer square with sides
+    of length 4. Both have points uniformally distributed around
+    the average.
+    """
+    def zip_arrays(d1,d2):
+        return np.array(list(zip(d1,d2)))
+
+    def make_squares(num, l):
+        left_right, top_bottom = make_barriers(num,l), make_barriers(num,l)
+        top_bottom[:,[0,1]] = top_bottom[:,[1,0]] 
+        square = np.concatenate((left_right, top_bottom))
+        return square
+
+    def make_barriers(num, l):
+        dist = 0.1*l
+        left = np.random.uniform((-l-dist),(-l+dist),num)
+        right = np.random.uniform((l-dist),(l+dist),num)
+        left_barrier = np.random.uniform(-l,l,num)
+        right_barrier = np.random.uniform(-l,l,num)
+        L = zip_arrays(left,left_barrier)
+        R = zip_arrays(right,right_barrier)
+        barriers = np.vstack((L,R))
+        return barriers
+    
+    outer = make_squares(num, l=2)
+    inner = make_squares(num, l=1)
+    frames = np.concatenate((outer,inner))
+    
+    half = int((len(frames)/2))
+    target = np.concatenate((np.zeros(half),np.ones(half)))
+    
+    X,y = shuffle(frames, target, random_state=1337)
+    return X,y
+
+
+def make_omega(n_episodes,p,q,r,scale,loc=0):
```

Comment:
I have no idea what omega, p, q and r mean.
It would be great if you could find a more descriptive name. 
If not (e.g. that's what they used in the paper and there is no easy interpretation) please at least add some explanation in the function description.

Reply:
Actually I also don't know what "loc" is ;)

Reply:
The name is true to what's in the paper, so I think `omega` should still be used. I'll leave a more informative docstring on how to interpret this. `p`, `q`, and `r` I define with comments in `main()`, but I can include descriptions in the functions as well.